README.md

```diff
@@ -388,42 +317,3 @@ needs to include all of the certificate-specific processing – in particular,
 checking that an item that has been suggested for inclusion is indeed a valid
 certificate that chains to an accepted root.
 
-### Verifiable Log-Backed Map
-
-One useful application for Trillian in Map mode is to provide a verifiable
-log-backed map, as described in the
-[Verifiable Data Structures](docs/papers/VerifiableDataStructures.pdf) white
-paper (which uses the term 'log-backed map').  To do this, a mapper personality
-would monitor the additions of entries to a Log, potentially external, and would
-write some kind of corresponding key:value data to a Trillian Map.
-
-Clients of the log-backed map are then able to verify that the entries in the
-Map they are shown are also seen by anyone auditing the Log for correct
-operation, which in turn allows the client to trust the key/value pairs
-returned by the Map.
-
-A concrete example of this might be a log-backed map that monitors a
-Certificate Transparency Log and builds a corresponding Map from domain names
-to the set of certificates associated with that domain.
-
-The following table summarizes properties of data structures laid in the
```

Comment:
It's a bit difficult to give it context, though? Keeping just that, it's kind of talking about the properties of something that doesn't appear to exist? Not quite sure where this should be going...

Reply:
It really just avoids having to dig into the verifiable data structures pdf.

Reply:
This is just a copy/paste from the (now quite old) "verifiable data structures" white paper, and I'm not even sure that we stand by those? For example, this doesn't mention consistency proofs, which is why we've effectively abandoned "online" maps in favour of researching batch ones...

Reply:
I think this info isn't useful in the main readme file. Maybe move it to `docs`, or indeed remove if it's outdated?

@mhutchinson Maybe you can point to more up-to-date docs?

Reply:
Yeah, the "in the main README file" is a bit part of this, IMHO. This information is *already* in `docs`, and it might have to be removed, potentially, but that would be something separate from this PR, for sure.

Reply:
I'm happy for this to go. We could link to the `batchmap` stuff, but I suggest that we add that separately.

Reply:
SGTM, thanks!